.ci/packaging.groovy

```diff
@@ -1,16 +1,18 @@
 #!/usr/bin/env groovy
 
-@Library('apm@current') _
+@Library('apm@test-build-error') _
 
 pipeline {
   agent none
   environment {
+    REPO = 'beats'
```

Comment:
NIT
```suggestion
    REPO = 'beats'
    BASE_DIR = "src/github.com/elastic/${env.REPO}"
```

Reply:
I'm ok with this change, although we do not use that variable. Is still needed?

Reply:
I don't understand, do you mean `BASE_DIR`? If so, it's used in the `gitCheckout`, if you mean `REPO` then it's used in https://github.com/elastic/beats/pull/21100/files#diff-bdaa5a61de14efda6eb2bbb2a38fa9a8R262

My NIT, was to avoid the hardcoded `beats` but it's not high important

Reply:
I meant the BASE_DIR, or at least I was not aware that the packaging job used it. We can keep it, even if it's just for consistency